src/pages/App.js

```diff
@@ -1,13 +1,23 @@
-import MainSvg from '../components/MainSvg';
-import './App.css';
+import { log } from '../lib/logger';
+import MainVisualization from '../components/MainVisualization';
+import coverageChecker from '../lib/coverageChecker/coverageChecker';
+import testbundle from '../data/fullBundle.json';
+import styles from './App.module.css';
+import MainSvg from '../components/oldSvg/MainSvg';
 
 function App() {
+  const coverageData = coverageChecker(testbundle);
+  log(coverageData);
```

Comment:
Is there a reason for us to be logging the coverage object?

Reply:
Mostly for debugging right now. We can keep it off of main for now, or I can add a TODO tag above it so that we remove it in the future. Let me know if you have a strong preference.